browser/resources/settings/brave_overrides/about_page.js

```diff
@@ -3,9 +3,21 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at http://mozilla.org/MPL/2.0/.
 
-import {RegisterPolymerTemplateModifications} from 'chrome://brave-resources/polymer_overriding.js'
+import {html,RegisterStyleOverride,RegisterPolymerTemplateModifications} from 'chrome://brave-resources/polymer_overriding.js'
```

Comment:
Please put some spaces between the imports (`html, RegisterStyleOverride, Re...`)

Reply:
Formatted the code @petemill Kindly review.